lib/srv/db/mysql/mysql_revoke_roles.sql

```diff
@@ -0,0 +1,22 @@
+CREATE PROCEDURE teleport_revoke_roles(IN username VARCHAR(32))
+BEGIN
+    DECLARE role VARCHAR(32) DEFAULT '';
```

Comment:
Is there 32 character limit on role names too? If yes, then this might be a problem in other places.

Reply:
Yes. Role has the same 32-char limit as user. Roles are created/maintained by the "admins" and provided to Teleport though. What problems do you foresee?

Reply:
I think adding this check would be a good idea. In non-strict mode, MySQL will silently truncate strings that are too long, which can lead to unpleasant results in our case.